tcms/testruns/views.py

```diff
@@ -38,94 +38,96 @@
+
+                test_run = TestRun.objects.create(
+                    product_version=test_plan.product_version,
+                    stop_date=None,
+                    summary=form.cleaned_data.get('summary'),
+                    notes=form.cleaned_data.get('notes'),
+                    plan=test_plan,
+                    build=form.cleaned_data['build'],
+                    manager=form.cleaned_data['manager'],
+                    default_tester=default_tester,
+                )
 
-            loop = 1
-            for case in form.cleaned_data['case']:
                 try:
-                    tcp = TestCasePlan.objects.get(plan=test_plan, case=case)
-                    sortkey = tcp.sortkey
+                    assignee_tester = User.objects.get(username=default_tester)
                 except ObjectDoesNotExist:
-                    sortkey = loop * 10
-
-                test_run.add_case_run(case=case,
-                                      sortkey=sortkey,
-                                      assignee=assignee_tester)
-                loop += 1
-
-            return HttpResponseRedirect(
-                reverse('testruns-get', args=[test_run.run_id, ])
-            )
-
-    else:
-        form = NewRunForm(initial={
-            'summary': 'Test run for %s' % test_plan.name,
-            'manager': test_plan.author.email,
-            'default_tester': request.user.email,
-            'notes': '',
-        })
-        form.populate(product_id=test_plan.product_id)
+                    assignee_tester = None
+
+                loop = 1
+                for case in form.cleaned_data['case']:
+                    try:
+                        tcp = TestCasePlan.objects.get(plan=test_plan, case=case)
+                        sortkey = tcp.sortkey
+                    except ObjectDoesNotExist:
+                        sortkey = loop * 10
+
+                    test_run.add_case_run(case=case,
+                                          sortkey=sortkey,
+                                          assignee=assignee_tester)
+                    loop += 1
+
+                return HttpResponseRedirect(
+                    reverse('testruns-get', args=[test_run.run_id, ])
+                )
 
-    context_data = {
-        'test_plan': test_plan,
-        'test_cases': tcs_values,
-        'form': form,
-    }
-    return render(request, 'testruns/mutable.html', context_data)
+        else:
+            form = NewRunForm(initial={
+                'summary': 'Test run for %s' % test_plan.name,
+                'manager': test_plan.author.email,
+                'default_tester': request.user.email,
+                'notes': '',
+            })
+            form.populate(product_id=test_plan.product_id)
+
+        context_data = {
+            'test_plan': test_plan,
+            'test_cases': tcs_values,
+            'form': form,
+        }
+        return render(request, 'testruns/mutable.html', context_data)
```

Comment:
Move the template name into a `template_name` class attribute. This will make it easier to transition to TemplateView or FormView later if we decide to.